src/Compilers/CSharp/Portable/Lowering/LocalRewriter/LocalRewriter_ForEachStatement.cs

```diff
@@ -358,129 +383,6 @@ private BoundExpression SynthesizeCall(CSharpSyntaxNode syntax, BoundExpression
             }
         }
 
-        /// <summary>
-        /// Lower a foreach loop that will enumerate the characters of a string.
-        ///
-        /// string s = x;
-        /// for (int p = 0; p &lt; s.Length; p = p + 1) {
-        ///     V v = (V)s.Chars[p];   /* OR */   (D1 d1, ...) = (V)s.Chars[p];
-        ///     // body
-        /// }
-        /// </summary>
-        /// <remarks>
-        /// We will follow Dev10 in diverging from the C# 4 spec by ignoring string's 
-        /// implementation of IEnumerable and just indexing into its characters.
-        /// 
-        /// NOTE: We're assuming that sequence points have already been generated.
-        /// Otherwise, lowering to for-loops would generated spurious ones.
-        /// </remarks>
-        private BoundStatement RewriteStringForEachStatement(BoundForEachStatement node)
```

Comment:
Nit: I noticed that `RewriteStringForEachStatement` is mentioned in a comment at the top of the file. Would be nice to fix that up.